content/building/environment-variables.md

```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
 4. Click **Save** to save the settings.
 
 ![](../uploads/env_vars.PNG)
+
+## Accessing environment variables from your application
+
+The next examples show how to place your Google Maps API key into an Android or iOS application from an environment variable. Doing it allows you not to store your secret key in the source code.
```

Comment:
the next > the following..
Doing it allows you not to... > With this approach you will not have to store your secret key in the repository.